db/migrate/20230308042434_backfill_auto_bump_cooldown_category_setting.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class BackfillAutoBumpCooldownCategorySetting < ActiveRecord::Migration[7.0]
```

Comment:
Sorry can I trouble you to add a spec here for the review? We don't have to commit it but it'll be nice if we can ensure that the migration is doing what it intends to do with a test. You can use the following template too run the migration within an rspec test.

```
# frozen_string_literal: true

require Rails.root.join(
  "db/migrate/20221201024458_make_channel_slugs_unique_with_index.rb",
)

RSpec.describe MakeChannelSlugsUniqueWithIndex do
  it "does whatever the migration is supposed to do" do
    MakeChannelSlugsUniqueWithIndex.new.up
  end
end
```

Reply:
Seems perfectly reasonable to me. 🙂

Reply:
@tgxworld [Added](https://github.com/discourse/discourse/pull/20507/files#diff-e72d2fe232e2cd88ae210379fcbfc330cc45960cec15b0553a7db690c3d2319bR5).

Reply:
Hmm I ran it locally and it fails for me as well. Maybe try to reset your test DB locally?

Reply:
After `RAILS_ENV=test rails db:reset db:migrate` it still passes. 😕 

```
bundle exec rspec spec/migrations/backfill_auto_bump_cooldown_category_setting_spec.rb      

Randomized with seed 35015
-- execute("INSERT INTO\n  category_settings(\n    category_id,\n    auto_bump_cooldown,\n    created_at,\n    updated_at\n  )\nSELECT\n  id,\n  1,\n  NOW(),\n  NOW()\nFROM categories\nON CONFLICT (category_id)\nDO\n  UPDATE SET\n    auto_bump_cooldown = 1,\n    updated_at = NOW();\n")
   -> 0.0004s
.

Finished in 0.26009 seconds (files took 1.65 seconds to load)
1 example, 0 failures
```

Reply:
Okay. Problem was solved by `RAILS_ENV=test rails db:drop db:create db:migrate`.